src/main/java/de/retest/recheck/cli/subcommands/Diff.java

```diff
@@ -5,6 +5,7 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.util.List;
```

Comment:
Minor: This shouldn't be a separate commit, squash with previous (dad64c08f19940497dcaac93a0f10e28be1e48e4).